### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by Ravvit528** [GoldieLeGenie/Dragon-quest-3HD2D-Randomizer#1]
<bug>
・When you check a shiny object, what is available from the shiny object on the field becomes the item you just got and it says it is no longer available.
・The "Gold%" and "Items%" on the Area tab are reversed.
・"Nexus mods" has stopped publishing.

<request>
・Please add a setting to make only those that do not have the original purchase price a random price.
・Please also add item and Gold probability settings to Randomize Shine Object.
・Please create an ON-OFF check box for each item in Randomize Shop Items.

※I am Japanese and not good at English, so I use a translator. I'm sorry if there are any words that might offend you.

**Comment by GoldieLeGenie** [comment]
Thank you for your report. I'll check it and work on a fix

**Comment by Ravvit528** [comment]
Thank you!
and I just sent an additional request.
If you don't mind, I would appreciate it if you could add it

**Comment by GoldieLeGenie** [comment]
should be fixed for shiny object and items gold reversed, you can download V0.2 , and for all your request im working on it 👍

**Comment by Ravvit528** [comment]
nice work!!
I'll download it and play with it right away

However, it seems that "Nexus Mods" cannot continue to download…

**Comment by GoldieLeGenie** [comment]
have fun ! yes im currently talking with the support to fix the problem 👍

**Comment by Ravvit528** [comment]
Also, this is a very luxurious request, but...
Currently, if the key items needed to proceed with the scenario are obtained randomly, the event will not proceed unless you re-acquire them in their original location. When I get a key item, can I set it up so that it can be flagged so that I can shortcut the progress?
*For example, they obtain pepper from shops and jars and obtain ships without going through Robin's subjugation event in Baharata.

**Comment by GoldieLeGenie** [comment]
important items are not randomized like orbs keys etc and if you have it early you can probably skip those areas

**Comment by Ravvit528** [comment]
Yesterday, I played from the beginning until the game was cleared,
Important items have come out of shops and jars many times. Orb I got it earlier, but I couldn't dedicate Orb to Lamia eggs without going through the original Orb acquisition event.

I also got a mirror of Lah in the early stages, but I didn't see the option of pointing the mirror at the king until I got a non-randomized mirror in a cave south of Samanosa.
